red_black_cards.py

```diff
@@ -0,0 +1,44 @@
+#Problem Statement: "You are given a standard deck of cards with 26 black cards and 26 red cards. You're now going to play a game: 
+#in every round, you can take one of two actions: you can stop playing, or you can draw another card. If you draw another card and 
+#it's red, you get one dollar. If you draw another card and it's black, you lose one dollar. What is the expected value of this game?"
+
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+import math
+
+try:
+    print("Input value cannot be cast to an int")
+    exit()
+
+try:
+    red = int(input('How many red cards are there?\n'))
+except:
+    print("Input value cannot be cast to an int")
+    exit()
+
+def expected_value(int: b, int: r) -> int:
+    "If either r or b is negative, then we cannot return a valid expected value"
```

Comment:
These comments are incorrectly formatted.

The correct format for comments is `# Comment here...`

The reason this code isn't giving you an error is because you're creating a valid string object and then doing nothing with it -- this string is a valid Python statement. Just like how in Python, you can have a single line that says `5`,  because `5` is a valid Python statement